README.rst

```diff
@@ -101,15 +101,17 @@ Usage Example
     while ticks_less(ticks_ms(), deadline):
         pass
 
+Documentation
+=============
+
+API documentation for this library can be found on `Read the Docs <https://docs.circuitpython.org/projects/neopixel/en/latest/>`_.
```

Comment:
The docs link should be this one I think:

https://docs.circuitpython.org/projects/ticks/en/latest/

Reply:
Whoops, that slipped through, thanks!